src/table/mod.rs

```diff
@@ -354,7 +358,7 @@ impl Table {
         result_string
     }
 
-    /// Cleans all the cards from the table
+    /// Cleans all the cards from_server_url the table
```

Comment:
Reckless replace all smh

Reply:
Oopsie

Reply:
I blame rust rover for giving me too much power